### ISSUE TITLE
Preserving colors in a pager + shell integration

### ISSUE THREAD

**Issue by mrzv** [zellij-org/zellij#2000]
The ability to open the scrollback in an editor is great, but it loses colors (out of necessity I believe). It would be great if there was an option to open the scrollback in a pager (e.g., `less -R`) that would preserve the colors.

It would be even more amazing if zellij supported something like [kitty's shell integration](https://sw.kovidgoyal.net/kitty/shell-integration/), where only the output of the last command could opened in a pager/editor.

**Comment by serranomorante** [comment]
A workaround for me is making my scrollback editor to interpret the `.dump` file as a different filetype. 
![image](https://github.com/zellij-org/zellij/assets/43413954/e23e731d-ed16-49b4-b731-506aac3255b1)

Example of `.dump` file with filetype `bash` on Neovim editor.
![image](https://github.com/zellij-org/zellij/assets/43413954/fcfb2085-a439-450f-89be-399c7c7af0cd)

It will never be the exactly same highlight as in the terminal, but is better than nothing.

**Comment by musjj** [comment]
Yeah, I really miss being able to directly jump to the top of the output of the last shell command. It's really useful when a command dumps pages of output at you.
